### PR TITLE
hub: update version to 2.12.1  - build/install manpages

### PR DESCRIPTION
**Pull request by ferdy**
#### Description

up

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/ondir/Portfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             ondir
+version          0.2.3
+revision         1
```
Comment (mf2k) [inline sysutils/ondir/Portfile]: The revision should start at 0.

File: sysutils/ondir/Portfile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+categories       sysutils
+platforms        darwin
+license          GLP-2
```
Comment (mf2k) [inline sysutils/ondir/Portfile]: I am assuming that the `license` is actually GPL-2?

File: sysutils/ondir/Portfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+maintainers      nomaintainer
+
+description      Automatic script execution on cd
+long_description OnDir is a program that automatically executes scripts as you traverse directories at a terminal.
+
+homepage         https://swapoff.org/ondir.html
+master_sites     https://swapoff.org/files/ondir/
+distfiles        ${name}-${version}.tar.gz
```
Comment (mf2k) [inline sysutils/ondir/Portfile]: Please do not set `distfiles` directly. Set the `distname` instead. However, this looks like the default. So this line should be deleted.

File: sysutils/ondir/Portfile (hunk 1 above)
Comment (ferdy) [reply sysutils/ondir/Portfile]: ciao @mf2k! thanks for your revision.
i fixed the revision number.

File: sysutils/ondir/Portfile (hunk 2 above)
Comment (ferdy) [reply sysutils/ondir/Portfile]: that's correct.
it is actually the content of the COPYING file (which is installed in the share/doc director).

File: sysutils/ondir/Portfile (hunk 3 above)
Comment (ferdy) [reply sysutils/ondir/Portfile]: removed the line! thanks again for your help! 
\--

**Comment by ferdy** [comment]
> Please clean the history / rebase changes against master.

ciao @mojca, it should be ok now. thanks!
\--
ferdy

**Comment by mf2k** [comment]
The commit message needs to be modified to conform to our standards. 

Something like this:
```
hub: update version to 2.12.1
 - build/install manpages

```

**Pull request merged**